### PR TITLE
Add Backers & Sponsors from OpenCollective

### PR DESCRIPTION
**Pull request by gr2m**
merge before publishing hoodiehq/hood.ie#302

[Preview](https://github.com/hoodiehq/hoodie/blob/a28641cb5f23419e2eb1a0c172683851e24fbeae/README.md) 👀

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,6 +105,66 @@ You can [read more about testing Hoodie](test).
...
+[![](https://opencollective.com/hoodie/backer/24/avatar)](https://opencollective.com/hoodie/backer/24/website)
+[![](https://opencollective.com/hoodie/backer/25/avatar)](https://opencollective.com/hoodie/backer/25/website)
+[![](https://opencollective.com/hoodie/backer/26/avatar)](https://opencollective.com/hoodie/backer/26/website)
+[![](https://opencollective.com/hoodie/backer/27/avatar)](https://opencollective.com/hoodie/backer/27/website)
+[![](https://opencollective.com/hoodie/backer/28/avatar)](https://opencollective.com/hoodie/backer/28/website)
+[![](https://opencollective.com/hoodie/backer/29/avatar)](https://opencollective.com/hoodie/backer/29/website)
+
+## Official Sponsors
+
+Show your support for Hoodie and [help us sustain our inclusive community](http://hood.ie/blog/sustaining-hoodie.html). We will publicly appreciate your and are happy to get your word out, as long as it aligns with our [Code of Conduct](http://hood.ie/code-of-conduct/).
```
Comment (varjmes) [inline README.md]: "publicly appreciate your" -> "publicly appreciate you"

**Comment by varjmes** [comment]
Can we get a screenshot of how this looks?

**Comment by gr2m** [comment]
you can preview the file when you go to the `files changed` tab and then click on the view button:
https://github.com/hoodiehq/hoodie/blob/a28641cb5f23419e2eb1a0c172683851e24fbeae/README.md

File: README.md (hunk 1 above)
Comment (gr2m) [inline README.md]: oh I wanted to say `publicly appreciate your support`, is that okay?

**Comment by varjmes** [comment]
Whoops, forgot this was a README.md change not a website change.

File: README.md (hunk 1 above)
Comment (varjmes) [inline README.md]: good to me!

**Comment by varjmes** [comment]
LGTM

**Comment by HipsterBrown** [comment]
LGTM 👍

**Pull request merged**